docs/contributing.rst

```diff
@@ -64,25 +64,31 @@ Ready to contribute? Here's how to set up :ref:`specutils <specutils>` for local
 
     $ git clone git@github.com:your_name_here/specutils.git
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+3. Install your local copy, preferably into some sort of virtual environment using your
+   preferred environment manager. For example, using conda::
 
-    $ mkvirtualenv specutils
+    $ conda create --name specutils-dev python pip
+    $ conda activate specutils-dev
     $ cd specutils/
-    $ python setup.py develop
+    $ pip install -e .[test]
 
 4. Create a branch for local development::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
    Now you can make your changes locally.
 
-5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox::
+5. When you're done making changes, check that your changes pass flake8 and the tests::
+
+    $ tox -e codestyle
+    $ pytest
```

Comment:
So, `pytest` didn't work for me. I had to run `tox` with the python-version specific config file specified.  If `pytest` _should_ work, I'm ok leaving this, but just heads up.

Reply:
Running `pytest` should work, and if it doesn't currently should be fixed. Are there further problems after the PR you opened the other day?